caas/kubernetes/provider/k8swatcher.go

```diff
@@ -8,12 +8,13 @@ import (
 
 	jujuclock "github.com/juju/clock"
 	"github.com/juju/errors"
+	"github.com/juju/juju/core/watcher"
```

Comment:
move me, and the tools/cache one below

Reply:
Not sure what you mean here. Looks fine to me, do you have an example?